pkgs/development/tools/kustomize/default.nix

```
@@ -1,29 +1,29 @@
-{ lib, buildGoModule, fetchFromGitHub }:
+{ lib, buildGoModule, fetchFromGitHub, tree }:
 
 buildGoModule rec {
   pname = "kustomize";
-  version = "3.1.0";
-  # rev is the 3.1.0 commit, mainly for kustomize version command output
-  rev = "95f3303493fdea243ae83b767978092396169baf";
+  version = "3.3.1";
+  # rev is the 3.3.1 commit, mainly for kustomize version command output
+  rev = "f2ac5a2d0df13c047fb20cbc12ef1a3b41ce2dad";
 
-  goPackagePath = "sigs.k8s.io/kustomize";
-  subPackages = [ "cmd/kustomize" ];
-
-  buildFlagsArray = let t = "${goPackagePath}/v3/pkg/commands/misc"; in ''
+  buildFlagsArray = let t = "sigs.k8s.iokustomize/v3/provenance"; in ''
```

@jonringer sorry there is a typo. I can't write when I'm outside of my editor...

```suggestion
  buildFlagsArray = let t = "sigs.k8s.io/kustomize/v3/provenance"; in ''
```

---

that was it, thought i misspelled provenance, so i checked that like 4x, then care to look at the rest of it.... lol

```
[13:08:36] jon@jon-desktop /home/jon/projects/nixpkgs (bump-kustomize)
$ ./result/bin/kustomize version
Version: {Version:3.3.1 GitCommit:f2ac5a2d0df13c047fb20cbc12ef1a3b41ce2dad BuildDate:unknown GoOs:linux GoArch:amd64}
```

---

well yeah this was quite a puzzle for me as well. That's the advertised simplicity of golang in practice :speak_no_evil:

---

pretty sure this is because this particular repo is doing a bit of hack to do this, I'm assuming most would just have a hardcoded string that they bump before doing a release cut